scripts/start.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash -ex
 
-curl --insecure -X POST -H "Content-Type: application/json" -d "@jupyter.json" https://minikube.lsst.codes/cachemachine/
-curl --insecure -X POST -H "Content-Type: application/json" -d "@simple.json" https://minikube.lsst.codes/cachemachine/
+curl --insecure -X POST -H "Content-Type: application/json" -d "@jupyter.json" https://minikube.lsst.codes/moneypenny/
+curl --insecure -X POST -H "Content-Type: application/json" -d "@simple.json" https://minikube.lsst.codes/moneypenny/
```

Comment:
I think you wanted to just delete this file. It doesn't seem relevant to you.

Reply:
done